components/BottomMenu.js

```diff
@@ -20,7 +20,7 @@ function BottomMenu ({ navigation }) {
     const [destination, setDestination] = React.useState("");
     const [mapPressed, setmapPressed] = React.useState("");
 
-    CurrentLocation();
+    CurrentBuildingInformation();
```

Comment:
I don't like the function being called here. Can we call it somewhere else? maybe in Map.js

Reply:
You are right it makes more sense !